com.wickeditor.WickEditor.yml

```diff
@@ -1,11 +1,11 @@
 app-id: com.wickeditor.WickEditor
 runtime: org.freedesktop.Platform
-runtime-version: "22.08"
+runtime-version: "23.08"
 sdk: org.freedesktop.Sdk
 base: org.electronjs.Electron2.BaseApp
-base-version: "22.08"
+base-version: "23.08"
 sdk-extensions:
-  - org.freedesktop.Sdk.Extension.node14
+  - org.freedesktop.Sdk.Extension.node18
```

Comment:
You need to change append-path too `append-path: /usr/lib/sdk/node18/bin`

Reply:
Argh! How did I forget that? :man_facepalming: 

I'll deal with it now.